graylog2-server/src/test/java/org/graylog/plugins/cef/parser/CEFMappingTest.java

```diff
@@ -18,9 +18,16 @@
 
 import org.junit.Test;
 
+import java.math.BigInteger;
+
 import static org.junit.Assert.assertEquals;
 
 public class CEFMappingTest {
+    private static final String[] BIG_INT_FIELD_KEYS = new String[]{
+            "cn1", "cn2","cn3", "cn4", "cnt", "destinationTranslatedPort", "dpid", "dpt",
+            "dvcpid", "fsize", "in", "oldFileSize", "out", "sourceTranslatedPort", "spid", "spt"
```

Comment:
Just a nit, but instead of converting the CEFMapping from the string values, you could just create an array of CEFMapping and skip that step in the test.

```suggestion
    private static final CEFMapping[] BIG_INT_FIELD_KEYS = new CEFMapping[]{
        CEFMapping.cn1, CEFMapping.cn2, CEFMapping.cn3, ..., CEFMapping.spt
```